rest-assured-tests/src/test/java/tariffManagementService/TariffTests.java

```diff
@@ -0,0 +1,121 @@
+package tariffManagementService;
+
+import commonTestClasses.FailedResponse;
+import org.junit.*;
+import org.junit.experimental.categories.Category;
+import org.junit.runners.AllTests;
+
+import java.util.ArrayList;
+
+import static io.restassured.RestAssured.*;
+import static org.junit.Assert.assertEquals;
+import static tariffManagementService.Tariff.*;
+
+public class TariffTests {
+    private static int tariffsNumber;
+    private static ArrayList<Integer> tariffsListToBeSaved = new ArrayList<>();
+    private final static String TARIFF_NAME_TO_DELETE = "NewTestTariffToDelete";
+    private final static String VALID_TARIFF_NAME = "NewTestTariff";
+    private final static String INVALID_TARIFF_NAME = "TariffWillNeverExist";
+    private final static String TARIFF_NAME_TO_DUPLICATE = "DuplicatedTariffName";
+    private final static float VALID_TARIFF_PRICE = 4.67f;
+    private final static int VALID_TARIFF_WASHING_TIME = 12;
+    private final static float INVALID_TARIFF_PRICE = -1.24f;
+    private final static int INVALID_TARIFF_WASHING_TIME = -7;
+    private final static int DELTA = 0;
+    private final static String INVALID_PARAMETER_MESSAGE = "Tariff not created, price and washing time should be a non negative";
```

Comment:
Is this exact text from tariff-service's responce?

Reply:
yes, it is